dockerfiles/ci/buster/0001-Better-support-for-cross-compilation.patch

```diff
@@ -0,0 +1,288 @@
+From f1ad9199eff32dd805bb5b155002bdb6ca69bf01 Mon Sep 17 00:00:00 2001
```

Comment:
Just out of interest, and so there's a record somewhere ...

Is this included in any version upstream ? If not, why not ?

Reply:
This patch has been applied to 8.1, but not to 8.0 on upstream. We only need it for cross-compiling 8.0. So maybe something which we can backport directly to 8.0...